packages/frontend/web-ui/src/common/helpers/validateNumberOfPlayers.ts

```diff
@@ -0,0 +1,17 @@
+import { Either } from '../models/Either';
+
+export const validateNumberOfPlayers = (
+  numberOfPlayers: number,
+): Either<string, undefined> => {
+  if (isNaN(numberOfPlayers)) {
```

Comment:
This logic is not enough to validate the number of players.

Hint: `Nan <= NUMBER_PLAYERS_MAXIMUM` evaluates to `false`.

Reply:
I've changed the validation.